vectorbender.py

```diff
@@ -103,28 +105,25 @@ def determineTransformationType(self):
             0 if no pairs Found
             1 if one pair found => translation
             2 if two pairs found => linear
-            3 if two pairs found => linear
```

Comment:
I think this was mislabeled in the original code (should have been `3 if three pairs found => affine, 4 if four or more pairs found => bending`), but must be kept for affine transformation to still work.

Reply:
Done!